metaseq/modules/transformer_layer.py

```diff
@@ -261,6 +262,12 @@ def __init__(
         )
         self.normalize_before = args.decoder_normalize_before
 
+        def floating_point_precision_convertor(x):
```

Comment:
Nit: move this to a utils file somewhere to not have two copies of the same function defined in two places.